src/site/stages/build/process-cms-exports/tests/transformer-helpers.unit.spec.js

```diff
@@ -17,6 +18,18 @@ describe('CMS export transformer helpers', () => {
     });
   });
 
+  describe('combineItemsInIndexedObject', () => {
+    it('turns an index-keyed object into an array', () => {
+      const obj = {
+        1: ['world'],
```

Comment:
The keys in the test data are strings. Because it's more fun this way. :stuck_out_tongue_winking_eye: 
```suggestion
        '1': ['world'],
```